lib/stdio/handle.js

```diff
@@ -21,40 +21,45 @@ export const handleInput = (addProperties, options) => {
 // For example, `stdout: ['ignore']` behaves the same as `stdout: 'ignore'`.
 const getStdioStreams = (stdioOption, index) => {
 	const optionName = getOptionName(index);
-	const stdioOptions = Array.isArray(stdioOption) ? [...new Set(stdioOption)] : [stdioOption];
```

Comment:
This PR basically changes `[...new Set(stdioOption)]` to the `filterDuplicates()` method, which takes into account generators.

Since `filterDuplicates()` requires knowing the type of each `stdioOptions` item, the logic had to be shuffled around a little. It does not change the behavior otherwise.